samples/benchmarks/coremark/boards/nrf5340dk_nrf5340_cpuapp.overlay

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ #include "../app.overlay"
```

Comment:
Move the button and LEDs configuration to a dedicated file at the board level: `app_aliases_common.overlay` and include it here and in the app.overlay.

Reply:
Don't understand you :) Please explain it in more detail :)

Reply:
My comment was quite detailed. Can you be more specific with your question? Which part of my comment don't you understand?

Reply:
What do you mean by " button and LEDs configuration"

Reply:
```
 / {
	aliases {
		button = &button0;
		led = &led0;
	};
 };
```